openshift/openshift/src/main/java/org/arquillian/cube/openshift/impl/client/OpenShiftAssistant.java

```diff
@@ -160,4 +170,37 @@ public DeploymentConfig deploymentConfig() {
             .withName(this.applicationName)
             .get();
     }
+
+    public OpenShiftAssistant usingTemplate(String templateURL) {
+        this.templateURL = templateURL;
+        return this;
+    }
+
+    public OpenShiftAssistant parameter(String param, String value) {
```

Comment:
Question? What's happen if user sets same parameter twice? Yes I know that both are inserted in the list, is this something ok, or we should replace the previous parameter with the new one like happens with HashMap?

@bartoszmajsak

Reply:
Behavior-wise it's the same I guess? Last one wins. However I would use a map and only convert to `ParameterValue`s when really used.